SQL-compiler/src/main/java/org/dbsp/sqlCompiler/circuit/SqlRuntimeLibrary.java

```diff
@@ -184,6 +184,8 @@ public String toString() {
     
     public FunctionDescription getFunction(
             String op, @Nullable DBSPType expectedReturnType, DBSPType ltype, @Nullable DBSPType rtype, boolean aggregate) {
+        if (ltype.is(DBSPTypeAny.class) || (rtype != null && rtype.is(DBSPTypeAny.class)))
+            throw new RuntimeException("Unexpected type _ for operand of " + op);
```

Comment:
Going forward, it'll help to have our own Exception types rather than throwing RTE.

Reply:
We have a few exception types, but I am not sure here it's beneficial.
Certainly they must extend Runtime, because otherwise they wreck havoc with the function signatures and lambdas.

Reply:
Yes, they have to extend RuntimeException or Exception, but having our own types makes it clear what type of error they are.

Reply:
You can file an issue for that. We have an issue about the compiler giving ugly error messages (exceptions instead of human-readable output).